### PR TITLE
Enhancement osm importer (road intersections + barriers)

### PR DESCRIPTION
**Pull request by DavidMansolino**
Linked with omichel/webots#3934

### CODE REVIEW

**Comment by DavidMansolino** [comment]
:+1:

**Comment by omichel** [comment]
:+1:

File: automobile/openstreetmap-importer.md [hunk 1]
```diff
@@ -23,19 +23,22 @@ You can use several arguments with this script:
...
+| --noTree            | Does not include the isolated trees in the generated world                                                                           | By default, isolated trees are included                                   |
+| --noBarrier         | Does not include the barriers (fence, wall, etc.) in the generated world                                                             | By default, barriers are included                                         |
+| --noRiver           | Does not include the rivers in the generated world                                                                                   | By default, rivers are included                                           |
+| --noBuilding        | Does not include the buildings in the generated world                                                                                | By default, buildings are included                                        |
+| --noRoadIntersection| Does not generate complex road intersections                                                                                         | By default, complex road intersections are generate                       |
```
Comment (stefaniapedrazzi) [inline automobile/openstreetmap-importer.md]: typo 'are generate' -> 'are generated'

Comment (DavidMansolino) [inline automobile/openstreetmap-importer.md]: The comment appeared on my screen at the exact same moment I merged, I will fix this in a new PR.

**Pull request merged**